test/correctness/document-correctness.py

```diff
@@ -213,60 +217,69 @@ def check_query(query, collection1, collection2, projection=None, sort=None, lim
         return False
 
 
-def test_update(collections, verbose=False):
+def test_update(collection1, collection2, verbose=False):
     okay = True
+    skip_current_iteration = False
```

Comment:
Minor nitpick, we don't seem to set `okay` and `skip_current_iteration` anywhere other than before `return`. It could just be removed instead, and use constant boolean values. Slightly more readable code.

Reply:
693f9eb